tests/test_user.py

```diff
@@ -458,3 +461,446 @@ def test_user_settings_edit_dev_app(self, driver, session, fake):
         finally:
             # Lastly use the api to delete the dev app as cleanup
             osf_api.delete_user_developer_app(session, app_id=app_id)
+
+
+@markers.dont_run_on_prod
```

Comment:
For future work: any plan to enable part or all of the tests to prod? PAT can be created, deleted and used without using up guid and piling up garbage in DB. One blocker here is the scopes are different I think.

Reply:
Our usual policy is not to create any new data in Production. Just for safety since these tests run unattended at night and we don't want anything to go crazy and break Prod. Not really worth the risk.

Reply:
Make sen. There is still a chance that token or developer app deletion may fail and leave database "corrupted".